src/Files.Backend/ViewModels/Shell/Tabs/TabItemViewModel.cs

```diff
@@ -25,6 +25,22 @@ public IconModel TabIcon
             set => SetProperty(ref _TabIcon, value);
         }
 
+        private bool isLoading;
+
+        public bool IsLoading
+        {
+            get => isLoading;
+            set => SetProperty(ref isLoading, value);
+        }
+
+        private bool allowStorageItemDrop;
+
+        public bool AllowStorageItemDrop
```

Comment:
Some of the properties are redundant like this one, so I'd suggest adding only the **required** ones and adding others later. Also, please stick to the new code style (`_IsLoading`, no empty line between the field and property)

Reply:
@d2dyno1 Thanks! Is there a particular reason why this property was deemed redundant? I will certainly remove it if that's the case.

Reply:
It's always true (I might be wrong though)
![image](https://user-images.githubusercontent.com/53011783/153705673-eec6b1ae-0028-4998-b3a1-bff82842a0d6.png)